File: lil-projects/text-inverter.lua
--- @author Josélio de S. C. Júnior <joseliojrx25@gmail.com>
--- @copyright Josélio de S. C. Júnior 2021

--- Reverses a string.
---@param x string Some string to be converted.
local function reversed(x) return string.reverse(x) end

-- In use
local text = 'Hello world!';
print(reversed(text)); -- Expected output: !dlrow olleH

File: lil-projects/text-inverter.js
/**
 * @author Josélio de S. C. Júnior <joseliojrx25@gmail.com>
 * @copyright Josélio de S. C. Júnior 2021
 */

/**
 * Reverses a string.
 * @param {string} x Some string to be converted.
 */
const reversed = x => x.split('').reverse().join('');

// In use
const text = 'Hello world!';
console.log(reversed(text)); //Expected output: !dlrow olleH